infrasim/model.py

```diff
@@ -1772,8 +1772,9 @@ def init(self):
 
 
     def get_commandline(self):
-        ipmi_cmd_str = "{0} -c {1} -f {2} -n -s /var/tmp".\
-            format(self.__bin, self.__config_file, self.__emu_file)
+        path = os.path.join(config.infrasim_home, self.__node_name, "data")
+        ipmi_cmd_str = "{0} -c {1} -f {2} -n -s {3}" .\
+            format(self.__bin, self.__config_file, self.__emu_file, path)
```

Comment:
CBMC can `get_workspace()`, so you don't have to compose path from `config.infrasim_home` and `self.__node_name`, which means, you don't have to change `infrasim/ipmi.py`

(I sincerely wan't to remove `infrasim/ipmi.py` someday...)